.github/workflows/docker-build-db.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Publish DB Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - docker-build-publish
+      - publish-docker-db
```

Comment:
Just commenting as a reminder that we might want to remove the extra branches before merging